data/distro.yml

```diff
@@ -570,3 +570,38 @@
         <p>To complete setup, restart your system. Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
       </li>
     </ol>
+
+
+- name: "Pardus"
+  logo: "pardus.svg"
+  info:  >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>A flatpak package is available in Pardus 2019 and newer. To install it, run the following as root:</p>
+        <pre><code>
+          <span class="unselectable">#</span> apt install flatpak
```

Comment:
Any reason not to recommend `sudo` to make the instructions easier?

Reply:
It has the same instructions with debian due to their base is debian

Reply:
`sudo` isn't recommended for most flatpak commands but there's no reason not to use it with `apt` commands. But alright, we can keep it the same as Debian.